kevm-pyk/src/kevm_pyk/kproj/evm-semantics/word.md

```diff
@@ -557,6 +558,7 @@ Range of types
     rule #rangeNonce    (   X          ) => #range ( 0               <= X < maxUInt64        )
     rule #rangeSmall    (   X          ) => #range ( 0               <= X < 10               )
     rule #rangeBlockNum (   X          ) => #range ( 0               <= X <= maxBlockNum     )
+    rule #rangeNegUInt64(   X          ) => #range ( pow56           <= X < pow64            )
```

Comment:
if this is intended to tell you whether a UInt64 when cast to an Int64 is negative, it is implemented wrong. The sign of an integer is a bit, not a whole byte.

Reply:
Not necessarily, the idea is actually to see if a `word` of UInt64 is negative. This was inspired by this `sgn` function: https://github.com/runtimeverification/evm-semantics/blob/bfd9e29ce15203b4db5f823ad362d0ab6a8b30ca/kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm-types.md?plain=1#L45-L50